Blog cml dedicated runner part 2

RCdeWit opened the pull request:
CML workflow is pending this change: https://github.com/iterative/dvc/pull/7213. If it gets merged before publication, we can clean up the workflow a little bit.

Otherwise this draft already incorporates the feedback I got earlier when it was still included in part 1. I am somewhat hesitant about the extent to which we should describe setting up the Google Drive remote. This is already covered in other resources; duplication might not be a great idea.

Thoughts?

jendefig commented on content/blog/2022-04-15-training-and-saving-models-on-a-dedicated-runner-part-2.md:
```diff
@@ -0,0 +1,236 @@
+  - Google Drive
+---
+
+In part 1 of this guide we showed how you can use CML to provision an AWS EC2
```
Will need to put link back to Part 1 article

jendefig commented on content/blog/2022-04-15-training-and-saving-models-on-a-dedicated-runner-part-2.md:
```diff
@@ -0,0 +1,236 @@
+- [Create a GCP service account](https://dvc.org/doc/user-guide/setup-google-drive-remote#using-service-accounts)
+- [Add `GDRIVE_CREDENTIALS_DATA` as a GitHub secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets)
+- [Create a Google Drive folder to save your model to](https://support.google.com/drive/answer/2375091?hl=en&co=GENIE.Platform%3DDesktop)
+- Grant the service account edit permissions to the Drive folder by sharing it
```
Would be good to have a link for this too.

jendefig commented on content/blog/2022-04-15-training-and-saving-models-on-a-dedicated-runner-part-2.md:
```diff
@@ -0,0 +1,236 @@
+training. We could, for example, use our runner to import our training data from
+a table in our database and write both the data and the model to the DVC remote.
+This is beyond the scope of this guide, but
+[here you can find a tutorial for this](https://github.com/iterative/cml_dvc_case).
```
Not sure "tutorial" is the right word for that.  Maybe "steps" or "instructions". Our audience loves repos.  Maybe better to say, "here you can find a repo with a template and steps for this."

jendefig reviewed (changes requested):
Looks great!  Just put a few minor things to be adjusted.

flippedcoder commented on content/blog/2022-05-09-CML-runners-saving-models-2.md:
```diff
@@ -0,0 +1,240 @@
+can look towards another one of Iterative's open-source tools:
+[DVC](https://dvc.org). DVC includes a lot of features for managing machine
+learning projects, such as ML pipelines and experiment tracking. In this guide
+we will zoom in on just one of those features: data versioning.
```
Data versioning is mentioned here, but it seems like the focus of the post is storing the model in a remote. Maybe this could be reworded to say we'll use the remote feature of DVC to store large models?

flippedcoder commented:
Could we include a link to the repo for this post, maybe in the prerequisites?

jendefig reviewed (approved):
Just these couple things then good  to go!

shcheklein commented on content/blog/2022-05-06-CML-runners-saving-models-2.md:
```diff
@@ -0,0 +1,243 @@
+  provisions an AWS EC2 instance to retrain a model and save the output on
+  a regular basis. In this part 2 we cover how to export the model to a DVC remote on Google Drive.
+picture: 2022-05-06/saving-models-2-cover.jpeg
+# pictureComment: Some _Comment_ (supports _basic_ [Markdown](link))
```
let's remove this please (no dead code)

shcheklein commented on content/blog/2022-05-06-CML-runners-saving-models-2.md:
```diff
@@ -0,0 +1,243 @@
+description:
+  Use CML to automatically retrain a model on a provisioned AWS EC2 instance and
+  export the model to a DVC remote storage on Google Drive.
+descriptionLong: |
```
use `>` instead and it will auto format it to <80 symbols per line

`|` is needed is when we do multiple paragraphs, etc ... we have to manually keep this section formatted (<80)

shcheklein commented on content/blog/2022-05-06-CML-runners-saving-models-2.md:
```diff
@@ -0,0 +1,243 @@
+  Use CML to automatically retrain a model on a provisioned AWS EC2 instance and
+  export the model to a DVC remote storage on Google Drive.
+descriptionLong: |
+  We can use CML to automatically retrain models whenever data, model code,
```
Do we want to start with a note and a link here that this is part two, part one is here ...

shcheklein commented:
Checkboxes in the description - are they all done? :)

shcheklein reviewed (commented):
Please folks, see a few stylistic changes that we need to take care of.

shcheklein commented on content/blog/2022-05-06-CML-runners-saving-models-2.md:
```diff
@@ -0,0 +1,243 @@
+
+# Conclusions
+
+As we saw in part 1 of this guide, we can use CML to automate a periodical
```
put a link I think?

RCdeWit commented:
Addressed these things here, @shcheklein: https://github.com/iterative/dvc.org/pull/3521
To be honest, not sure if I'm doing this the right way; should I have reverted the merge instead?

shcheklein commented:
@RCdeWit hey, no need to revert anything :) Just a new PR to do these small changes is enough.
